knora/dsplib/utils/excel_to_json_lists.py

```diff
@@ -11,19 +10,59 @@
 from openpyxl import load_workbook
 from openpyxl.cell import Cell
 from openpyxl.worksheet.worksheet import Worksheet
+from knora.dsplib.models.helpers import BaseError
 
-# Global variables used to ensure that there are no duplicate node names
 list_of_lists_of_previous_cell_values: list[list[str]] = []
+"""Module level variable used to ensure that there are no duplicate node names"""
+
 list_of_previous_node_names: list[str] = []
+"""Module level variable used to ensure that there are no duplicate node names"""
+
 
+def expand_lists_from_excel(
+    lists_section: list[dict[str, Union[str, dict[str, Any]]]]
+) -> list[dict[str, Any]]:
+    """
+    Checks if the "lists" section of a JSON project file contains references to Excel files. Expands all Excel files to
+    JSON, and returns the expanded "lists" section. If there are no references to Excel files, the "lists" section is
+    returned as is.
+
+    Args:
+        lists_section: the "lists" section of a parsed JSON project file. If this is an empty list, an empty list will be returned.
 
-def get_values_from_excel(
+    Returns:
+        the same "lists" section, but without references to Excel files
+    """
+    new_lists = []
+
+    for _list in lists_section:
+        if "folder" not in _list["nodes"]:
+            # this list is a JSON list: return it as it is
+            new_lists.append(_list)
+        else:
+            # this is a reference to a folder with Excel files
+            prepared_rootnode, excel_file_names = _prepare_list_creation(
+                excelfolder=_list["nodes"]["folder"],
+                listname=_list["name"],
+                comments=_list["comments"]
+            )
+            print("\tThe following Excel files will be temporarily expanded into the 'lists' section of your project:")
+            [print(f"\t - {filename}") for filename in excel_file_names]
+
+            finished_list = _make_json_list_from_excel(prepared_rootnode, excel_file_names, verbose=False)
+            new_lists.append(finished_list)
+
+    return new_lists
+
+
+def _get_values_from_excel(
     excelfiles: dict[str, Worksheet],
     base_file: dict[str, Worksheet],
     parentnode: dict[str, Any],
     row: int,
     col: int,
-    preval: list[str]
+    preval: list[str],
+    verbose: bool = False
```

Comment:
This method is used 
 - by the `create` command, where it shouldn't print too much by default
 - during the `excel` command, where it should print every created node by default

The `verbose` switch seemed to be a good solution for this.